instance/models/openedx_instance.py

```diff
@@ -99,6 +146,10 @@ def save(self, **kwargs):
             self.use_ephemeral_databases = settings.INSTANCE_EPHEMERAL_DATABASES
         if not self.edx_platform_commit:
             self.edx_platform_commit = self.openedx_release
+        if not self.s3_bucket_name:
+            self.s3_bucket_name = self.swift_container_name.replace('_', '-')
```

Comment:
Could you use a configurable prefix this too, like [for the IAM user](https://github.com/open-craft/opencraft/pull/295/files#diff-b66ef4710ed40f1b980e3ab3a34ec812R126)? Would be convenient for humans browsing S3.

Reply:
@pomegranited do you think this should be a field in the database like self.s3_bucket_prefix, or are you thinking on an extra setting?